src/pages/LoginAndRegister/GuestAccountClaim.vue

```diff
@@ -30,7 +30,15 @@
 						Enter last name.
 					</template>
 				</kv-base-input>
-				<kv-button class="claim-button smaller" type="submit">
+				<kv-button
+					class="claim-button smaller"
+					type="submit"
+					v-kv-track-event="[
```

Comment:
I think it would be better to put this at the start of the `claimGuestAccount` function, in case someone submits the form by pressing enter instead of clicking the submit button.

Reply:
Makes sense, good catch.